source/_patterns/basics/01-buttons/icon-button.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Icon Button
+---
+## Overview
+
+A button containing an icon is used as part of the [Patternfly toolbar](https://www.patternfly.org/pattern-library/forms-and-controls/toolbar/#_design) and in other places.
```

Comment:
I was pointing to that since there is no specific design document for this element. What else would be good to put in the overview?